draft-scurtescu-secevent-simple-control-plane.md

```diff
@@ -244,7 +239,7 @@ Host: transmitter.example.com
 Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
 
 {
-  "phone_number": "123-456-7890"
+  "phone_number": "+1 206 555 1234"
```

Comment:
Nit: example phone numbers should be in the range 555-01XX:
https://en.wikipedia.org/wiki/555_(telephone_number)

(I never knew about this until now)